extensions/fairifier-rdf-extension/scripts/rdf-schema-alignment.js

```diff
@@ -118,7 +116,45 @@ RdfSchemaAlignmentDialog.prototype._constructBody = function(body) {
                         self._nodeUIs = [];
                         self._renderBody(body);
                         DialogSystem.dismissUntil(self._level);
+                      } else {
+                        return;
                       }
+                    },
+                    function(rdfSkeleton) {
+                      $.getJSON("command/core/get-operations?" + $.param({
+                        project: theProject.id
+                      }), function(json) {
+                        var operations_old = json.entries.map(function(val) {
+                          return val['operation'];
+                        });
+
+                        if (operations_old.length == 1) {
+                          operations_old = [];
+                        } else {
+                          operations_old = operations_old.slice(-5);
```

Comment:
Why only the last 5? Seems like such an arbitrary number.

Reply:
We need to discuss what we want to keep but 5 seems the smallest amount of provenance that still works